src/schema.py

```diff
@@ -1,107 +1,36 @@
+from graphene.types.datetime import Date
 import pytz
 import requests
 
 from src.constants import *
+from src.types import (
+    AccountInfoType,
+    EateryType,
+    TransactionType,
+)
 
 class Data(object):
   eateries = {}
-  events = {}
-  menus = {}
 
   @staticmethod
-  def update_data(**kwargs):
-    Data.eateries = kwargs.get('eateries')
-    Data.events = kwargs.get('events')
-    Data.menus = kwargs.get('menus')
-
-class CoordinatesType(ObjectType):
-  latitude = Float(required=True)
-  longitude = Float(required=True)
-
-class CampusAreaType(ObjectType):
-  description = String(required=True)
-  description_short = String(required=True)
-
-class PaymentMethodsType(ObjectType):
-  brbs = Boolean(required=True)
-  cash = Boolean(required=True)
-  cornell_card = Boolean(required=True)
-  credit = Boolean(required=True)
-  mobile = Boolean(required=True)
-  swipes = Boolean(required=True)
-
-class DiningItemType(ObjectType):
-  category = String(required=True)
-  description = String(required=True)
-  healthy = Boolean(required=True)
-  item = String(required=True)
-  show_category = Boolean(required=True)
-
-class FoodItemType(ObjectType):
-  item = String(required=True)
-  healthy = Boolean(required=True)
-  sort_idx = Int(required=True)
-
-class FoodStationType(ObjectType):
-  category = String(required=True)
-  items = List(FoodItemType, required=True)
-  item_count = Int(required=True)
-  sort_idx = Int(required=True)
-
-class EventType(ObjectType):
-  cal_summary = String(required=True)
-  description = String(required=True)
-  end_time = String(required=True)  # <isodate>:<time>
-  menu = List(FoodStationType, required=True)
-  start_time = String(required=True)  # <isodate>:<time>
-  station_count = Int(required=True)
-
-class OperatingHoursType(ObjectType):
-  date = String(required=True)
-  events = List(EventType, required=True)
-  status = String(required=True)  # so far, we've only seen status = 'EVENT'
-
-class EateryType(ObjectType):
-  about = String(required=True)
-  about_short = String(required=True)
-  campus_area = Field(CampusAreaType, required=True)
-  coordinates = Field(CoordinatesType, required=True)
-  eatery_type = String(required=True)
-  id = Int(required=True)
-  image_url = String(required=True)
-  location = String(required=True)
-  name = String(required=True)
-  name_short = String(required=True)
-  operating_hours = List(OperatingHoursType, required=True)
-  payment_methods = Field(PaymentMethodsType, required=True)
-  phone = String(required=True)
-  slug = String(required=True)
-
-class TransactionType(ObjectType):
-  name = String(required=True)
-  timestamp = String(required=True)
-
-class AccountInfoType(ObjectType):
-  brbs = String(required=True)
-  cityBucks = String(required=True)
-  history = List(TransactionType, required=True)
-  laundry = String(required=True)
-  swipes = String(required=True)
+  def update_data(eateries):
+    Data.eateries = eateries
```

Comment:
no more events or menus?

Reply:
I think events (`EventType`) got moved to `types/operating_hours_type.py` and menus (`List(FoodStationType)`) got moved to `types/food_station_type.py`.

Reply:
they're never queried, everything is from `eateries`